packages/kuma-gui/mk/install.mk

```diff
@@ -8,5 +8,6 @@
 
 .PHONY: .clean
 .clean:
-	find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +
+	@echo "Recursively removing all node_modules/ directories in `pwd`..."; \
+		find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +
```

Comment:
Oh one thing I just noticed, ideally `find` would always execute in the workspace root, no matter where I use `make` from. I'd be happy doing this in any sort of follow up though, so no action here just a note.